app/src/pages/authoring/[resourceType]/[id].tsx

```diff
@@ -16,11 +16,28 @@ export default function ResourceAuthoringPage() {
 
   const ctx = trpc.useContext();
 
-  const resourceQuery = trpc.draft.getDraftById.useQuery({
+  const { data: resource } = trpc.draft.getDraftById.useQuery({
```

Comment:
this causes a trpc error when the page first loads, as `resourceType` will be undefined on first load. Maybe allow for the `getDraftbyId` input to have resourceType as optional, and if it's not provided we just return null or something